_posts/2018-06-05-june.md

```diff
@@ -26,16 +26,13 @@ meetupdate: 2018-06-05
     {% comment %} ==== TALK / PRESENTATION - SLOT #2 ====     {% endcomment %}
 
     {% comment %} TITLE OF YOUR TALK:                         {% endcomment %}
-                  <h4>Free Slot</h4>
-
-    {% comment %} SUB-TITLE OR DESCRIPTION:                   {% endcomment %}
-                  <p>This 25 minute slot could be yours!</p>
-
-    {% comment %} REMOVE NEXT LINE:                           {% endcomment %}
-                  <p><strong>Submit your talk now</strong></p>
+        <h4>Squeezing PHPUnit</h4>
 
-    {% comment %} UNCOMMENT/EDIT NEXT LINE:                   {% endcomment %}
-    <!-- <p><a class="speaker" target="_blank" href="https://twitter.com/...HOW-TO-CONTACT-YOU">YOUR NAME</a></p> -->
+   {% comment %} SUB-TITLE OR DESCRIPTION:                   {% endcomment %}
```

Comment:
I think you are actually supposed to remove these comments, @keradus!

🤓

Reply:
there are not removed in previous talk, nor in previous event, so I left them as-is ;) no big deal here ;)